cmake/kokkos_tpls.cmake

```diff
@@ -20,6 +20,8 @@ KOKKOS_TPL_OPTION(HWLOC   Off)
 KOKKOS_TPL_OPTION(LIBNUMA Off)
 KOKKOS_TPL_OPTION(MEMKIND Off)
 KOKKOS_TPL_OPTION(CUDA    Off)
+KOKKOS_TPL_OPTION(LIBRT   Off)
+KOKKOS_TPL_OPTION(LIBDL   On)
```

Comment:
We need to mirror this in `BUILD.md`.

Reply:
This (and other changes) are now in BUILD.md.